ccp/lib/utils/authorization.py

```diff
@@ -0,0 +1,24 @@
+"""
+This file contains authorization handlers
+"""
+
+
+class Authorization(object):
+
+    def __init__(self):
```

Comment:
shouldn't you be expecting some constructor params here?

Reply:
that you will add to `self.authorization_header_string` var ?

Reply:
Fixed